### DISCUSSION TITLE
I can not run build with NextJs

### DISCUSSION THREAD

**Discussion by minhth1529** [ant-design/ant-design#42902] [Q&A]
When I build with NextJs, antd 5.0.0, please help me.

![image](https://github.com/ant-design/ant-design/assets/17897146/21ff9dca-0ac3-4c5f-aac7-f29f79fabfa5)

**Comment by zombieJ** [comment]
Strange. Seems this def is exist here: https://unpkg.com/browse/antd@5.6.1/es/calendar/generateCalendar.d.ts

Is that clear all cache and reinstall tryed?

**Reply by minhth1529** [reply]
yes, i ran rm -fr node_modules && yarn && yarn build on nextjs already

**Reply by minhth1529** [reply]
I try to run: https://app.warp.dev/block/W34eMrWwI2JTn2nCuXHM8q

after that: https://app.warp.dev/block/sKN7mNkeZ4Tes87nO2VXaQ
